tests/setup.py

```diff
@@ -51,7 +51,7 @@ def run_tests(self):
       license="apache2.0",
       packages=find_packages(),
       include_package_data=True,
-      install_requires=["ducktape==0.8.14", "requests==2.24.0"],
+      install_requires=["ducktape==0.11.3", "requests==2.24.0"],
```

Comment:
please use `0.11.4` and the version of `requests` is `2.31.0`

Reply:
Oh, it seems like `confluent/ducktape` `0.11.4` does not exists in  Github Release, but it's at pypl.